smokey/src/main/java/frc/robot/control/ManualInputInterfaces.java

```diff
@@ -115,12 +117,11 @@ public double getInputEveryBotUptakeTrigger()
   {
     // use the co drivers right trigger
     double inputValue = 0.0;
-    // TODO - need help to know signs for both
     if(this.coDriverControllerGamePieceTarget == ChargedUpGamePiece.Cone) {
-      inputValue = coDriverController.getRightTriggerAxis() * -1.0;
+      inputValue = coDriverController.getRightTriggerAxis();
```

Comment:
similar changes for this file are in the other PR (the other PR has more changes so it might be confusing on merge conflict to keep these around)